crates/forge/tests/e2e/running.rs

```diff
@@ -759,12 +760,10 @@ fn available_gas_error() {
     let snapbox = test_runner();
 
     let output = snapbox.current_dir(&temp).assert().failure();
-    assert_stdout_contains!(
+    assert_stderr_contains!(
```

Comment:
Why did it only change here while other diagnostics are still printed to stdout?

Reply:
It is not a diagnostic, it is a custom error of ours. It comes from stderr of `scarb-snforge-test-collector` which we inherit.

In case of erros caught directly in forge we print them `fn main` in `forge/src/main.rs` to the stdout (don't really know why, I wanted to change it someday but remember we decided not to do it).

Reply:
It could be a weird from the perspective of use to show these errors like this, without an `[ERROR]` tag before the message or whatever. 

It may be fine if it's just `available_gas` but we should consider some approach of displaying the collector errors in manner more consistent with the rest of the forge.

Reply:
Minor thing imo, you can add an issue if you want

Reply:
Quite a change in UX in my opinion, and annotations and errors emitted by them are already limited.